packages/botonic-react/package.json

```diff
@@ -49,9 +49,14 @@
     "@types/react": "16.9.23",
     "babel-dts-generator": "^0.6.3",
     "babel-plugin-add-module-exports": "^1.0.2",
+    "eslint": "^6.8.0",
```

Comment:
@dpinol I'm gonna remove this in a different PR

Reply:
okay, as commented now all depends on the configuration of the parent. 👍